pkg/otlp/collector.go

```diff
@@ -95,6 +97,10 @@ func getBuildInfo() (component.BuildInfo, error) {
 type PipelineConfig struct {
 	// OTLPReceiverConfig is the OTLP receiver configuration.
 	OTLPReceiverConfig map[string]interface{}
+	// OpenCensusEnabled reports whether the OpenCensus receiver is enabled
```

Comment:
Can we add some comments, here or somewhere else, making it very clear that this feature is internal only and may be removed at any time in a minor version?

Reply:
Done.